APMrover2/Rover.h

```diff
@@ -125,16 +125,17 @@ class Rover : public AP_HAL::HAL::Callbacks {
     void setup(void) override;
     void loop(void) override;
 
+    // all settable parameters
+    Parameters g;
+    ParametersG2 g2;
```

Comment:
why this change ?

Reply:
If they're private I can't access frame_type parameter in AP_MotorsUGV.cpp.

Reply:
just make a getter : get_frame_type on rover class.